### DISCUSSION TITLE
OBS30 and Dante Virtual Soundcard (DVS)

### DISCUSSION THREAD

**Discussion by ITaluone** [obsproject/obs-studio#11560] [Development Questions]
Hi 
I have a question about using OBS >=30 with DVS: I OBS29 and prior OBS selected only channel 1&2 from a 32x32 channel DVS. This was quite useful (but maybe bug-using) to have the streaming device (mac-mini) have connected every channel (e.g. for concurrent audio recording), but only cares about channels 1&2.

With OBS>=30 this isn't possible anymore, since it listens to all 32 inputs and therefore we get (even when main channels 1&2 are muted) noise into the stream (from source 3-32).

Is there any way to tell OBS to "use" the old behavior? Or Maybe this could be a feature that one can manually select channels.

Thanks in advance 🙏

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!

**Reply by IT-VBFK** [reply]
Really, is it?

If something works in v29 but doesn't in v30.. sounds more like a regression to me. 🤔

**Reply by Fenrirthviti** [reply]
If it's a reproducible issue that can be confirmed to be a regression, than an issue should be opened with steps to replicate, not a discussion post anyway.

As it stands, one user reporting a third party solution not working isn't enough information for us to do anything.
